docs/sources/features/explore/index.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Explore"
+type = "docs"
+[menu.docs]
+name = "Explore"
+identifier = "explore"
+parent = "features"
+weight = 5
++++
+
+# Introduction
+
+One of the major new features of Grafana 6.0 is the new query-focused Explore workflow for troubleshooting and/or for data exploration.
+
+Grafana's dashboard UI is all about building dashboards for visualization. Explore strips away all the dashboard and panel options so that you can focus on the query. Iterate until you have a working query and then think about building a dashboard.
+
+For infrastructure monitoring and incident response, you no longer need to switch to other tools to debug what went wrong. Explore allows you to dig deeper into your metrics and logs to find the cause. The new logging feature in Explore correlates metrics and logs and allows for a tight integration in Grafana. This creates a new debugging workflow where you can:
+
+1. Receive an alert
+2. Drill down and examine metrics
+3. Drill down again and search logs related to the metric and time interval (and in the future, distributed traces).
+
+But if you just want to explore your data and do not want to create a dashboard then Explore makes this much easier. Explore will also if possible show both graph and table results.
```

Comment:
s/But/Even/

Reply:
Hmm, don't like that how this sentence reads. Will rewrite the whole sentence.

Reply:
Left out the But and rewrote the next sentence.